docs/advanced/api/rest.md

```diff
@@ -2,161 +2,189 @@
-        "forward_asset": "XCP",
-        "forward_quantity": 100000000,
-        "backward_asset": "BTC",
-        "backward_quantity": 800000,
-        "tx0_block_index": 310491,
-        "tx1_block_index": 310492,
-        "block_index": 310492,
-        "tx0_expiration": 2000,
-        "tx1_expiration": 2000,
-        "match_expire_index": 310512,
-        "fee_paid": 7200,
-        "status": "pending",
-        "MAX(rowid)": 1
-    },
-    ...
-]
-```
-
+FORMAT: 1A
+
+# Counterpaty Core API
+
+The Counterparty Core API is the recommended way to query the state of a Counterparty node. The following routes are available.
+
+# Assets
+
+Resources related to assets in the API.
+
+## Asset [/assets/{asset}]
+
+An `asset` object has the following attributes:
+
++ asset
++ asset_longname
++ description
++ divisible
++ issuer
++ locked
++ owne
++ supply
+
++ Parameters
+    + asset: "NOTDIVISIBLE" (required, string) - Name of the Asset
+
+### View an Asset Detail [GET]
+
++ Response 200 (application/json)
+
+        {
+            "asset": "NOTDIVISIBLE",
+            "asset_longname": None,
+            "description": "Indivisible asset",
+            "divisible": False,
+            "issuer": "mn6q3dS2EnDUx3bmyWc6D4szJNVGtaR7zc",
+            "locked": False,
+            "owner": "mn6q3dS2EnDUx3bmyWc6D4szJNVGtaR7zc",
+            "supply": 1000
+        }
+
+### View a Balances of a given Asset [/assets/{asset}/balances]
+
++ Response 200 (application/json)
+
+        [
+            {
+                "address": "1_mn6q3dS2EnDUx3bmyWc6D4szJNVGtaR7zc_mtQheFaSfWELRB2MyMBaiWjdDm6ux9Ezns_2",
+                "asset": "XCP",
+                "quantity": 300000000
+            },
+            {
+                "address": "2MyJHMUenMWonC35Yi6PHC7i2tkS7PuomCy",
+                "asset": "XCP",
+                "quantity": 46449548498
+            },
+            ...
+        ]
+
+### View Orders for a given Asset [/assets/{asset}/orders]
+
++ Response 200 (application/json)
+
+        [
+            {
+                "tx_index": 11,
+                "tx_hash": "1899b2e6ec36ba4bc9d035e6640b0a62b08c3a147c77c89183a77d9ed9081b3a",
+                "block_index": 310010,
+                "source": "mn6q3dS2EnDUx3bmyWc6D4szJNVGtaR7zc",
+                "give_asset": "XCP",
+                "give_quantity": 100000000,
+                "give_remaining": 100000000,
+                "get_asset": "BTC",
+                "get_quantity": 1000000,
+                "get_remaining": 1000000,
+                "expiration": 2000,
+                "expire_index": 312010,
+                "fee_required": 900000,
+                "fee_required_remaining": 900000,
+                "fee_provided": 6800,
+                "fee_provided_remaining": 6800,
+                "status": "open"
+            },
+            ...
+        ]
+
+# Addresses
+
+Resources related to addresses in the API.
+
+### View an Address Balances [/assets/{address}/balances]
```

Comment:
"View Balances of a Given Asset"

Reply:
Now it's "Get Balances By Address". This title is automatically generated from the function name `get_balance_by_address()`.